### PR TITLE
New inbox

### PR DESCRIPTION
**Pull request by upexcel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Hr_System project on ES6 RESTful Express API",
   "main": "dist",
   "scripts": {
-    "start": "node ./models/index.js && nodemon --max_old_space_size=8192 -w src --exec \"babel-node src --presets es2015,stage-0\"",
+    "start": "node ./models/index.js && nodemon --max_old_space_size=524288 -w src --exec \"babel-node src --presets es2015,stage-0\"",
```
Comment (reactexcel) [inline package.json]: why is this needed? does the app crash?

Comment (upexcel) [reply package.json]: yes it crashed once on local
